packages/Box/Box.jsx

```diff
@@ -30,6 +30,8 @@ const getBetweenClasses = (scale, inline, desktop) => {
 }
 
 /**
+ * @version 0.0.0-development
```

Comment:
I think we want `@version 1.0.0` here.... We are going to deploy these docs to prod when we release next, so might as well put version 1.0 now. Find + replace?